packages/codegen/src/ink-types.ts

```diff
@@ -208,6 +209,18 @@ export function generateInkTypes(lookup: InkMetadataLookup) {
                 value: callable.types.value,
                 docs: [],
               },
+              ...(callable.default
+                ? [
+                    {
+                      label: "default",
+                      value: {
+                        type: "inline",
+                        value: "true",
+                      },
+                      docs: [],
+                    } satisfies StructField,
+                  ]
+                : []),
```

Comment:
Can `mutates` & `payable` also be included in the descriptor?

Reply:
Mhh it might be a good idea. What for?

I imagine `payable` could be used to make "value" mandatory, but mutates?

(always keep in mind that descriptors are subject to change at any time, but I think it could make sense to add these properties if the ink client can benefit from them)

Reply:
I personally would use `mutates` to explicitly segregate view-only messages & mutating ones. Would provide better DX imo.

> always keep in mind that descriptors are subject to change at any time

Yeah I'm aware of this :p

Reply:
I've added both of them. They can't be used in the ink-client, but I reckon they can be useful for the ink-sdk.